packages/material-ui/src/Menu/Menu.js

```diff
@@ -149,18 +149,19 @@ const Menu = React.forwardRef(function Menu(inProps, ref) {
     event.preventDefault();
     setOpenSubMenuIndex(null);
     if (onClose) {
-      onClose(event, `${(event && event.key && event.key.toLowerCase()) || 'unknown'}KeyDown`);
```

Comment:
Why was this changed?

Reply:
Because I found it to be unnecessary now and it was also a little weird with that fallback case because you could get an event name of `unknownKeyDown` which is definitely not a MUI thing. It seems pretty safe since this only fires during a keydown event which should always have an event with a key on it to `toLowerCase()`.

Reply:
Cool.